clang/test/SemaSYCL/sycl-esimd.cpp

```diff
@@ -2,8 +2,9 @@
 
 // ----------- Negative tests
```

Comment:
Here too, please add a comment describing what this is testing.
Also, could you clean up this test to conform to “Guidelines for adding DPC++ in-tree LIT tests (DPC++ Clang FE tests)” in the https://github.com/intel/llvm/blob/sycl/CONTRIBUTING.md guide for the suggested guidelines?

Reply:
I would suggest cleaning up in a separate patch since it is unrelated to the changes in this patch and will distract attention from the important changes. I can open a new issue/task for this. @premanandrao, what you think?

Reply:
Done.